bulletin/index.html

```diff
@@ -190,7 +190,9 @@ <h1 class="article-banner--brand" itemprop="brand">MACILE</h1>
 				<div class=" article-content feature-article__content">
 					<h2 class="article-title"> Engineering Robust Learning Environments Through Continuous Improvemnt</h2>
 					<p class="article-abstract"> 
-							MACILE is a research-based program founded to advance <span>STEM</span> education in less advantaged communities, engineering solutions to optimally increase access to robust<span>language-infused STEM</span> learning environments. Development follows three phases: exploration, design refinement, and expansion. We review results from 2007-2008. Clearly, focus on continuous improvement is producing astoundingly encouraging results.
+						MACILE is a research-based program founded to advance <span>STEM</span> education in less advantaged communities, engineering solutions to optimally increase access to robust<span>language-infused STEM</span> learning environments. This work review design improvements from 2007-2008.
+					</p>
+					<p ><a class="btn-small--yellow" href="/bulletin/macile-model-development-2007-08-ef.pdf"></a> 
```

Comment:
HTML tags should not have extra spaces in the `<p>` not `<p >`.

You also added a PDF but the PDF was never added to the repo. I recommend putting the PDF at the root level in a folder called `content`.

Reply:
Done.

Did not realize the PDF file was untracked.